src/meta_schedule/schedule_rule/schedule_rule.cc

```diff
@@ -251,6 +251,33 @@ Array<ScheduleRule> ScheduleRule::DefaultHexagon() {
   };
 }
 
+Array<ScheduleRule> ScheduleRule::DefaultMicro() {
+  return {
+      ScheduleRule::AutoInline(
+          /*into_producer=*/false,
+          /*into_consumer=*/true,
+          /*inline_const_tensor=*/true,
+          /*disallow_if_then_else=*/true,
+          /*require_injective=*/true,
+          /*require_ordered=*/true,
+          /*disallow_op=*/Array<String>{"tir.exp"}),
+      ScheduleRule::MultiLevelTilingWideVector(
+          /*structure=*/"SRSRS",
+          /*vector_length_in_bits=*/1024,
```

Comment:
You probably copied this from Hexagon, but the normal `ScheduleRule::MultiLevelTiling` is probably what you want.

Reply:
Thanks for the comment. I updated the rule.